config/settings/common.py

```diff
@@ -360,7 +360,8 @@ def url_prefix_list(val: str) -> List[str]:
 WORKER_DYNO_NAME = env("WORKER_DYNO_NAME", default=None) or "worker"
 
 # Queues that the given app will has
-HEROKU_APP_QUEUES = env("HEROKU_APP_QUEUES", default=("high", "medium", "default"))
+queues = env.list("HEROKU_APP_QUEUES", default=["high", "medium", "default"])
+HEROKU_APP_QUEUES = set(queues)
```

Comment:
This should be a list; not a set. There's a line in the autoscaling code which assumes all workers are processing the first queue, and turning this into a set loses the ordering.